code/autolab/logging_utils.py

```diff
@@ -2,27 +2,30 @@
 import time
 from docker import from_env
 import os
-from .docker_utils import kill_if_running, blocking_start
+from docker import from_env
+from docker.errors import NotFound
+
+
+BAG_RECORDER = 'bag_recorder'
 
 
 def start_logging(filename, device_list, mount_folder):
     docker = from_env()
-    name = "bag_recorder"
 
     # try to remove another finish bag_recorder container
     try:
-        docker.containers.get(name).remove(force=True)
+        docker.containers.get(BAG_RECORDER).remove(force=True)
     except:
         pass
 
     # Create the log folder
-    print(mount_folder)
-    os.makedirs(mount_folder, exist_ok=True)
+    if not os.path.exists(mount_folder):
+        os.makedirs(mount_folder)
```

Comment:
@afdaniele  isnt the (exist_ok = true ) option better ?

Reply:
but the print statement should indeed go

Reply:
@AmaurX 
Let's try with exist_ok